schedule/yam/agama.yaml

```diff
@@ -1,7 +1,9 @@
 ---
 name: agama
 description: >
-  Perform interactive installation with agama.
+  Perform automatic installation testing with agama.
```

Comment:
the current description is ok, I would keep it, both interactive and unattended (or auto-) installations are automatic, we automate both of them. 'test' or 'testing' normally should be used, because it is all about that, most of the time doesn't give you anything else.

Reply:
Solved aca47525aabf6b10f5285e7a04d1d88eee7da4d4